### PR TITLE
FFWEB-1563: Added information about `channel` attribute in `ff-checkout-tracking`

### PR DESCRIPTION
**Pull request by a-laurowski**

### CODE REVIEW

File: markdown/3.x/en/documentation/tracking-guide.md [hunk 1]
```diff
@@ -99,7 +99,13 @@ If you want to prevent the auto requests, you can disable them with the Boolean
    <ff-checkout-tracking-item record-id="10321943" count="1"></ff-checkout-tracking-item>
 </ff-checkout-tracking>
 ```
-If you do so, you can poke around with the child elements and invoke the `checkoutTrackingElement.trackCheckoutItems()` function manually.
+If you want to send tracking request to the different channel, use `channel` attribute
```
Comment (wdutkiewicz-obj) [inline markdown/3.x/en/documentation/tracking-guide.md]: I am wondering: `to the different channel` vs `to a different channel`. The later option seems to be more accurate in this context. @omikron-fp: what do you make of it?

File: markdown/3.x/en/documentation/tracking-guide.md [hunk 2]
```diff
@@ -99,7 +99,13 @@ If you want to prevent the auto requests, you can disable them with the Boolean
...
+```html
+<ff-checkout-tracking disable-auto-tracking>
+   <ff-checkout-tracking-item record-id="10321926" count="5" channel="new-channel-name"></ff-checkout-tracking-item>
```
Comment (wdutkiewicz-obj) [inline markdown/3.x/en/documentation/tracking-guide.md]: [OPTIONAL]
I'd replace `new-channel-name` with something meaningful or at least wrap it with square brackets to stress that it os only a placeholder for some channel name.

File: markdown/3.x/en/documentation/tracking-guide.md [hunk 3]
```diff
@@ -99,7 +99,13 @@ If you want to prevent the auto requests, you can disable them with the Boolean
...
+</ff-checkout-tracking>
+```
+If you do so, you can search for the child elements and invoke the `checkoutTrackingElement.trackCheckoutItems()` function manually.
```
Comment (wdutkiewicz-obj) [inline markdown/3.x/en/documentation/tracking-guide.md]: [OPITONAL]
I think I have already replaced `poke around` with some more official expression. Please, check if there are more occurrences in the documentation and replace if applicable. A good chance to improve it a little.

File: markdown/1.x/en/documentation/tracking-guide.md [hunk 4]
```diff
@@ -98,7 +98,7 @@ If you want to prevent the auto requests, you can disable them with the Boolean
    <ff-checkout-tracking-item record-id="10321943" count="1"></ff-checkout-tracking-item>
 </ff-checkout-tracking>
 ```
-If you do so, you can poke around with the child elements and invoke the `checkoutTrackingElement.trackCheckoutItems()` function manually.
+If you do so, you can search for the child elements and invoke the `checkoutTrackingElement.trackCheckoutItems()` function manually.
```
Comment (omikron-fp) [inline markdown/1.x/en/documentation/tracking-guide.md]: Thank you!

File: markdown/3.x/en/documentation/tracking-guide.md (hunk 1 above)
Comment (omikron-fp) [reply markdown/3.x/en/documentation/tracking-guide.md]: I'd go for a bit more details.

_If you want to send the tracking request to a channel different from the one configured in `ff-communication`, use the `channel` attribute._

File: markdown/3.x/en/documentation/tracking-guide.md [hunk 5]
```diff
@@ -99,7 +99,13 @@ If you want to prevent the auto requests, you can disable them with the Boolean
    <ff-checkout-tracking-item record-id="10321943" count="1"></ff-checkout-tracking-item>
 </ff-checkout-tracking>
 ```
-If you do so, you can poke around with the child elements and invoke the `checkoutTrackingElement.trackCheckoutItems()` function manually.
+If you want to send the tracking request to a channel different from the one configured in ff-communication, use the `channel` attribute.
```
Comment (omikron-fp) [inline markdown/3.x/en/documentation/tracking-guide.md]: For consistency:
```suggestion
If you want to send the tracking request to a channel different from the one configured in `ff-communication`, use the `channel` attribute.
```
For even more consistency, there are several mentions of `ff-` elements in regular text that could be unwrapped from `< >`. But I'll leave that to you to decide.

File: markdown/3.x/en/documentation/tracking-guide.md [hunk 6]
```diff
@@ -99,7 +99,13 @@ If you want to prevent the auto requests, you can disable them with the Boolean
...
 </ff-checkout-tracking>
 ```
-If you do so, you can poke around with the child elements and invoke the `checkoutTrackingElement.trackCheckoutItems()` function manually.
+If you want to send the tracking request to a channel different from the one configured in `ff-communication`, use the `channel` attribute.
+```html
+<ff-checkout-tracking disable-auto-tracking>
+   <ff-checkout-tracking-item record-id="10321926" count="5" channel="[NEW_CHANNEL]"></ff-checkout-tracking-item>
+</ff-checkout-tracking>
+```
+If you do so, you can search for the child elements and invoke the `checkoutTrackingElement.trackCheckoutItems()` function manually.
```
Comment (omikron-fp) [inline markdown/3.x/en/documentation/tracking-guide.md]: I think this line makes more sense at its original position before `If you want to send the tracking request` (L102).
And while at it, the sentence could need a little update:
```suggestion
If you do so, you can search for the child elements by invoking the `checkoutTrackingElement.trackCheckoutItems()` function manually.
```

File: markdown/3.x/en/documentation/tracking-guide.md [hunk 7]
```diff
@@ -99,13 +99,14 @@ If you want to prevent the auto requests, you can disable them with the Boolean
    <ff-checkout-tracking-item record-id="10321943" count="1"></ff-checkout-tracking-item>
 </ff-checkout-tracking>
 ```
-If you want to send the tracking request to a channel different from the one configured in `ff-communication`, use the `channel` attribute.
+If you do so, you can search for the child elements by invoking the `checkoutTrackingElement.trackCheckoutItems()` function manually.
+
+If you want to send the tracking request to a channel different from the one configured in ff-communication, use the `channel` attribute.
```
Comment (omikron-fp) [inline markdown/3.x/en/documentation/tracking-guide.md]: Formatting is gone again.
```suggestion
If you want to send the tracking request to a channel different from the one configured in `ff-communication`, use the `channel` attribute.
```

**Pull request merged**